packages/forms-web-app/src/pages/projects/projects.router.js

```diff
@@ -78,4 +79,6 @@ if (featureFlag.allowSection51) {
 	router.use(section51Router);
 }
 
+router.get('/:case_ref/project-updates', middleware, getProjectUpdatesController);
```

Comment:
not certain about this, but shouldn't this be behind the `allowProjectInformation` flag similarily to the project information page, since they both touch same data?